tilequeue/queue/file.py

```diff
@@ -43,6 +43,9 @@ def read(self):
     def job_done(self, msg_handle):
         pass
 
+    def job_partially_done(self, handle):
```

Comment:
Is there a more descriptive name for this? Perhaps `job_keepalive` or `job_made_progress`, or something that indicates that it's called repeatedly to indicate liveness?

Reply:
Good idea. I liked `job_progress` and called it that in facde1f. Sound ok?